css/css-conditional/container-queries/var-evaluation.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<title>CSS Container Queries Test: var() in size query values</title>
+<link rel="help" href="https://drafts.csswg.org/css-conditional-5/#size-container">
+<link rel="help" href="https://drafts.csswg.org/mediaqueries-4/#units">
```

Comment:
Next time around, it probably doesn't hurt to also add a `<link rel="help" href="https://github.com/w3c/csswg-drafts/issues/8088">` pointing to the relevant github issue.  But I don't think we need to bother fixing it here.

Reply:
Let's see if the UI lets me do it without using anything on the command line, though.
```suggestion
<link rel="help" href="https://drafts.csswg.org/mediaqueries-4/#units">
<link rel="help" href="https://github.com/w3c/csswg-drafts/issues/8088">
```